docker/Dockerfile-controller

```diff
@@ -5,8 +5,8 @@ RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s
 RUN chmod u+x kubectl && mv kubectl /usr/local/bin/kubectl
 RUN curl -LO https://istio.io/operator.yaml
 RUN mkdir -p /usr/local/var/lib/aci-cni
-COPY pkg/istiocrd/upstream-istio-operator-1.4.0.yaml /usr/local/var/lib/aci-cni/upstream-istio-operator.yaml
-COPY pkg/istiocrd/upstream-istio-cr-1.4.0.yaml /usr/local/var/lib/aci-cni/upstream-istio-ctrlplane-resource.yaml
+COPY pkg/istiocrd/istioctl-1.5.2 /usr/local/bin/istioctl
```

Comment:
Is it an option to download istioctl like what we are doing with kubectl?

Reply:
@snaiksat good point; apparently there is a way and I fixed it.